### DISCUSSION TITLE
iOS Accessibility Support

### DISCUSSION THREAD

**Discussion by igroti** [lichess-org/mobile#409] [Ideas]
Are there any plans on supporting iOS accessibility features? Such as Voice Control, VoiceOver, Text Size, Differentiate Without Color.

Thanks

P.S.: I'm new here, if this is in the wrong category, just let me know.

0-0!#

**Comment by veloce** [comment]
Yes there are. For the issues don't exist but I'll create them soon to track the progress on this topic.
